third_party/common/dawn/dawn_wire/server/ObjectStorage.h

```diff
@@ -0,0 +1,228 @@
+// Copyright 2019 The Dawn Authors

```

Comment:
should we use this file in dawn?

Reply:
Yes, Most code are the same as dawn, but there are different namespace and `ContextInfo ` is used in webnn, but `DeviceInfo` for dawn in the file.

Reply:
It makes sense to fork it for now. What's the path of the original dawn file? Could we track we will refactory it somewhere?

Reply:
Add annotation of refactoring in the file.